membrane_curvature/base.py

```diff
@@ -35,8 +35,8 @@ class MembraneCurvature(AnalysisBase):
     select : str or iterable of str, optional. 
         The selection string of an atom selection to use as a
         reference to derive a surface.
-    pbc : bool, optional
-        Apply periodic boundary conditions.
+    wrap : bool, optional
+        Apply coordinate wrapping.
```

Comment:
I'd clarify what this means.  This will pack all the particles into the primary unit cell before processing them further right?

Reply:
Yes, correct, it's to keep all articles in the primary unit cell. I'll update this. Thanks.